### PR TITLE
Fixing indentation

### PR DESCRIPTION
**Pull request by manute**

### CODE REVIEW

File: src/lib/deployer.js [hunk 1]
```diff
@@ -191,17 +191,17 @@ class Deployer extends EventEmitter {
...
 					.done();
+				});
 			});
-		});
+		}
 	}
-}
 
-module.exports = Deployer;
+	module.exports = Deployer;
```
Comment (chesleybrown) [inline src/lib/deployer.js]: Why the indent down here?

Comment (manute) [inline src/lib/deployer.js]: ouch, I have installed `atom` for indent all the file and I don't know why it did that

Comment (chesleybrown) [inline src/lib/deployer.js]: That's a weird setting to have for sure... lol

**Comment by c1freitas** [comment]
LGTM ~ Thanks!

**Pull request merged**